plotly/tests/test_core/test_tools/test_figure_factory.py

```diff
@@ -1125,6 +1125,245 @@ def test_table_with_index(self):
                                                 'zeroline': False}}}
         self.assertEqual(index_table, exp_index_table)
 
+
+class TestGantt(TestCase):
+
+    def test_validate_gantt(self):
+
+        # validate the basic gantt inputs
+
+        df = [dict(Task='Job A',
+                   Start='2009-02-01',
+                   Finish='2009-08-30',
+                   Complete='a')]
```

Comment:
No big deal (not worth changing here), but if your just using `dict`, it's typically more common to use the literal notation `{}`.